packages/suite/src/components/suite/Preloader/__tests__/Preloader.test.tsx

```diff
@@ -449,7 +457,8 @@ describe(`${Preloader.name} component`, () => {
         const { unmount } = renderWithProviders(store, <Index app={store.getState().router.app} />);
 
         expect(findByTestId('@connect-device-prompt')).not.toBeNull();
-        expect(findByTestId(/TR_DEVICE_IN_BOOTLOADER/)).not.toBeNull();
+        expect(findByTestId('TR_DEVICE_CONNECTED_BOOTLOADER')).not.toBeNull();
+        fireEvent.click(findByTestId('@onboarding/troubleshooting-tips/button'));
```

Comment:
IMHO it's really bad practice to check hidden content, it was pretty hard to debug 🤷‍♂️

Now we click first before we check.